abstract-spec/MC_ffg.tla

```diff
@@ -94,11 +94,19 @@ VotesWellFormedInv ==
     /\ \A vote \in votes: 
         /\ vote.ffg_vote \in ffg_votes
         /\ vote.validator \in VALIDATORS
+
+JustifiedCheckpointsInv == 
     /\ \A c \in justified_checkpoints: 
         /\ IsValidCheckpoint(c)
-        \* Something about justified_checkpoints
-        \* /\ TODO
+        /\ IsJustified(c, votes, justified_checkpoints)
+    /\ LET allCheckpoints == {Checkpoint(block, i): block \in blocks, i \in CheckpointSlots} 
+       IN \A c \in (allCheckpoints \ justified_checkpoints): ~IsJustified(c, votes, justified_checkpoints)
+
+VoteAndCheckpointInv ==
+    /\ VotesWellFormedInv
+    /\ JustifiedCheckpointsInv
 
+InductiveInv == GraphInv /\ VoteAndCheckpointInv
```

Comment:
I cannot comment above, but can we sort the conjuncts in the body of `GraphInv` either alphabetically or in the order `GraphWellFormedInv`, `RealClosureInv`, `GraphIsTreeInv` are defined (or, ideally, both)?